pkg/process/util/containers/containers_test.go

```diff
@@ -414,6 +414,7 @@ func TestGetContainers(t *testing.T) {
 			},
 			ThreadCount: 10,
 			ThreadLimit: 20,
+			Tags:        []string{},
```

Comment:
Does this change only occur for tests, or does it occur with actual functionality as well (returning an empty slice instead of nil)?

Reply:
No that's only for the tests as the Mock implementation was not the same as the normal one.